tests/cephfs/nfs/test_nfs_mount.py

```diff
@@ -0,0 +1,202 @@
+        mounts_by_client = {client: [] for client in clients}
+        for i, command in enumerate(nfs_export_list):
+            client_index = i % nfs_client_count  # Cycle through clients
+            client = clients[client_index]
+            mounts_by_client[client].append(command)
+        log.info(f"Clients and respective exports:  {mounts_by_client}")
+        for client,mounts in mounts_by_client.items():
+
+            exports_to_mount = mounts[:exports_per_client]
+            for mount in exports_to_mount:
+                nfs_mounting_dir = f"/mnt/nfs_{mount.replace('/', '')}"
+                fs_util.cephfs_nfs_mount(client, nfs_server, mount, nfs_mounting_dir)
+                execution_time_cmd = fs_util.function_execution_time(client1.exec_command)
+                execution_time_cmd(
+                    sudo=True,
+                    cmd=f"mkdir -p {nfs_mounting_dir}/{mount};python3 /home/cephuser/smallfile/smallfile_cli.py "
+                    f"--operation create --threads 1 "
+                    f"--file-size 1024 "
+                    f"--files 1024 --top "
+                    f"{nfs_mounting_dir}/{mount}",
+                    long_running=True,
+                )
+        log.info("Test completed successfully")
+        end_time = datetime.now()
+        exec_time = (end_time - start_time).total_seconds()
+        log.info(
+            f"Iteration Details \n"
+            f"Export name created : {nfs_export_name}\n"                
+            f"Time Taken for the Iteration : {exec_time}\n"
+        )
+        writer.writerow(
+            {
+                "Export_name": nfs_export_name,
+                "Execution_time": exec_time,
+            }
+        )
+        csvfile.flush()
+        os.fsync(csvfile.fileno())
+        test_run_details.append(
+            {
+                "Export_name": nfs_export_name,
+                "Execution_time": exec_time,
+            }
+        )
+        return 0
+    except Exception as e:
+        log.info(e)
+        log.info(traceback.format_exc())
+        [
+            print(
+                "%s          %s\n"
+                % (item["Export_name"], item["Execution_time"])
+            )
+            for item in test_run_details
+        ]
+        log.info(traceback.format_exc())
+        return 1
+    finally:
+        log.info("Cleaning up")
+        log.info(test_run_details)
+        [
+            print(
+                "%s          %s\n"
+                % (item["Export_name"], item["Execution_time"])
+            )
+            for item in test_run_details
+        ]
+        status, rc = clients[0].exec_command(
+            sudo=True,
+            cmd="ceph -s",
+        )
+        df, rc = clients[0].exec_command(
+            sudo=True,
+            cmd="ceph df",
+        )
+        csvfile.write(status)
+        csvfile.write(df)
+        csvfile.flush()
+        csvfile.close()
+
+        fs_util.remove_nfs_export(
```

Comment:
Shouldn't the removal be looping over all the exports created and ensure they are deleted?